Detectors/ITSMFT/ITS/reconstruction/include/ITSReconstruction/CookedTracker.h

```diff
@@ -45,14 +46,19 @@ namespace its
 class CookedTracker
 {
   using Cluster = o2::itsmft::Cluster;
+  using Vertex = o2::dataformats::Vertex<o2::dataformats::TimeStamp<int>>;
 
  public:
   CookedTracker(Int_t nThreads = 1);
   CookedTracker(const CookedTracker&) = delete;
   CookedTracker& operator=(const CookedTracker& tr) = delete;
   ~CookedTracker() = default;
 
-  void setVertices(std::vector<std::array<Double_t, 3>>& vertices) { mVertices = std::move(vertices); }
+  void setVertices(std::vector<Vertex>& vertices)
```

Comment:
Why the tracker needs to own the vertices? Isn't it better just to pass ``const std::vector<Vertex>& vertices`` and assign it to a const pointer?